src/java.base/share/classes/jdk/internal/access/JavaIOFilePermissionAccess.java

```diff
@@ -44,5 +45,6 @@ public interface JavaIOFilePermissionAccess {
      * @return the new FilePermission using the alt path (as npath)
      *         or null if no alt path is available
      */
+    @SuppressWarnings("removal")
     FilePermission newPermUsingAltPath(FilePermission input);
```

Comment:
I assume JavaIOFilePermissionAccess can be removed, nothing should be using this now.

Reply:
Filed JDK-8354053 to remove as a separate issue.